common/flights/src/impls/session_api_impl.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020-2021 The Datafuse Authors.
+//
+// SPDX-License-Identifier: Apache-2.0.
+
+use common_exception::Result;
+pub use common_store_api::SessionApi;
+use common_tracing::tracing;
+
+use crate::RequestFor;
+use crate::StoreClient;
+use crate::StoreDoAction;
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug)]
+pub struct KillSessionReq {
+    pub session_id: String,
+}
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug)]
+pub struct KillSessionReply;
+
+#[async_trait::async_trait]
+impl SessionApi for StoreClient {
+    #[tracing::instrument(level = "debug", skip(self))]
+    async fn kill(&mut self, session_id: String) -> Result<()> {
```

Comment:
what is session_id? it's query_id in fusequery?

Reply:
oops, it should be `query_id`, let me rename it

Reply:
should I rename the method to `kill_query` as well?

Reply:
Yes, maybe `kill_query` is better.